src/services/adService.ts

```diff
@@ -0,0 +1,44 @@
+import { IAdModel } from '../db/';
+import { IAd } from '../interfaces/';
+import { getAll, save } from '../repositories/adRepository';
+
+export const getAllAds = async () => {
+  return await getAll();
+};
+
+export const updateAds = async (ads: IAd[]) => {
+  await saveAds(ads);
+};
+
+const updateAdsFields = async (knownAd: IAd, ads: IAd[]) => {
+  let selectedAd;
+  for (const ad of ads) {
+    if (knownAd.sourceUrl === ad.sourceUrl) {
```

Comment:
What is this? Don't we have .find method?

Reply:
fixed =) I totally forgot about [].find